Feature/jls 6210515296/new nci2i page

ecstech-admin opened the pull request:
## What type of PR is this? (check all applicable)

- [x] Feature
- [ ] Bug Fix
- [ ] Hot Fix
- [ ] Optimization
- [ ] Documentation Update

## Description
Develop newly designed NCI2I page

## Related Tickets & Documents
- Monday Task #6210515296
- https://blackberg-group.monday.com/boards/1931719392/pulses/6210515296

## QA Instructions, Screenshots, Recordings
New page: https://dev-innovation.bgsandbox.com/ecosystem/views/national-centers/
Compare developed page to Figma design
Confirm carousel, map and page, in general functions correctly at different break points

### UI accessibility checklist
_If your PR includes UI changes, please utilize this checklist:_
- [ ] Semantic HTML implemented?
- [ ] Keyboard operability supported?
- [ ] Checked with [Google Lighthouse](https://developer.chrome.com/docs/lighthouse/overview) and addressed `Critical` and `Serious` issues?
- [ ] Color contrast tested?


## Are there any post deployment tasks we need to perform?
- Add GA code on production
- Proper meta tags utilized 

## Changed Files in this Pull Request ##
- _includes/home/accelerating-hci.html
 - _includes/home/featured-successes.html
 - _includes/home/masthead.html
 - _includes/home/national-centers-map.html
 - _includes/home/national-centers.html
 - _includes/home/news-and-events.html
 - _includes/home/our-mission.html
 - _site/assets/css/map-svg/mapsvg.css
 - _site/assets/css/map-svg/nanoscroller.css
 - _site/assets/img/home/5g-tower.png
 - _site/assets/img/home/accelerating-innovation.png
 - _site/assets/img/home/drhci-logo.png
 - _site/assets/img/home/ideas-logo.png
 - _site/assets/img/home/ncchi-logo.png
 - _site/assets/img/home/nci2i-1.jpg
 - _site/assets/img/home/nci2i-10.jpg
 - _site/assets/img/home/nci2i-2.jpg
 - _site/assets/img/home/nci2i-3.jpg
 - _site/assets/img/home/nci2i-4.jpg
 - _site/assets/img/home/nci2i-5.jpg
 - _site/assets/img/home/nci2i-6.jpg
 - _site/assets/img/home/nci2i-7.jpg
 - _site/assets/img/home/nci2i-8.jpg
 - _site/assets/img/home/nci2i-9.jpg
 - _site/assets/img/home/nci2i-masthead.png
 - _site/assets/img/home/necie-logo.png
 - _site/assets/img/home/news/bring-people-ideas-together.jpg
 - _site/assets/img/home/news/capstone-project.jpg
 - _site/assets/img/home/news/disease-prevention-tools.jpg
 - _site/assets/img/home/news/face-of-innovation.jpg
 - _site/assets/img/home/news/first-innovation-center.jpg
 - _site/assets/img/home/news/jesse-award-osborne.jpg
 - _site/assets/img/home/news/legacy-partners.jpg
 - _site/assets/img/home/news/ncchi.jpg
 - _site/assets/img/home/news/personalized-healthcare.png
 - _site/assets/img/home/news/predictive-analytics.png
 - _site/assets/img/home/news/project-convergence.png
 - _site/assets/img/home/news/rideshare-for-veteran-employment.jpg
 - _site/assets/img/home/news/robert-jesse-award.jpg
 - _site/assets/img/home/news/smart-sock.jpg
 - _site/assets/img/home/news/va-health-incubator.png
 - _site/assets/img/home/news/veteran-infertility.jpg
 - _site/assets/img/home/news/veteran-rideshare.jpg
 - _site/assets/img/home/news/vulnerable-veterans-rideshare.jpg
 - _site/assets/img/home/news/xprize-pandemic-alliance.jpg
 - _site/assets/img/home/va-ventures-logo.png
 - _site/assets/img/home/vha-nci2i-logo.svg
 - _site/assets/img/map-svg/default-pin.svg
 - _site/assets/img/map-svg/usa.svg
 - _site/assets/js/mustache-fellows-render.js
 - _site/assets/js/mustache-nci2i-news-render.js
 - _site/assets/js/nci2i-home-news.js
 - _site/assets/js/vendors/lazy/jquery.lazy.min.js
 - _site/assets/js/vendors/map-svg/jquery.mousewheel.min.js
 - _site/assets/js/vendors/map-svg/jquery.nanoscroller.min.js
 - _site/assets/js/vendors/map-svg/mapsvg.functions.js
 - _site/assets/js/vendors/map-svg/mapsvg.min.js
 - _site/assets/sass/vha/pages/home/accelerating-hci/_index.scss
 - _site/assets/sass/vha/pages/home/featured-successes/_index.scss
 - _site/assets/sass/vha/pages/home/masthead/_index.scss
 - _site/assets/sass/vha/pages/home/national-centers-map/_index.scss
 - _site/assets/sass/vha/pages/home/national-centers/_index.scss
 - _site/assets/sass/vha/pages/home/news-and-events/_index.scss
 - _site/assets/sass/vha/pages/home/our-mission/_index.scss
 - assets/css/map-svg/mapsvg.css
 - assets/css/map-svg/nanoscroller.css
 - assets/img/home/5g-tower.png
 - assets/img/home/accelerating-innovation.png
 - assets/img/home/drhci-logo.png
 - assets/img/home/ideas-logo.png
 - assets/img/home/ncchi-logo.png
 - assets/img/home/nci2i-1.jpg
 - assets/img/home/nci2i-10.jpg
 - assets/img/home/nci2i-2.jpg
 - assets/img/home/nci2i-3.jpg
 - assets/img/home/nci2i-4.jpg
 - assets/img/home/nci2i-5.jpg
 - assets/img/home/nci2i-6.jpg
 - assets/img/home/nci2i-7.jpg
 - assets/img/home/nci2i-8.jpg
 - assets/img/home/nci2i-9.jpg
 - assets/img/home/nci2i-masthead.png
 - assets/img/home/necie-logo.png
 - assets/img/home/news/bring-people-ideas-together.jpg
 - assets/img/home/news/capstone-project.jpg
 - assets/img/home/news/disease-prevention-tools.jpg
 - assets/img/home/news/face-of-innovation.jpg
 - assets/img/home/news/first-innovation-center.jpg
 - assets/img/home/news/jesse-award-osborne.jpg
 - assets/img/home/news/legacy-partners.jpg
 - assets/img/home/news/ncchi.jpg
 - assets/img/home/news/personalized-healthcare.png
 - assets/img/home/news/predictive-analytics.png
 - assets/img/home/news/project-convergence.png
 - assets/img/home/news/rideshare-for-veteran-employment.jpg
 - assets/img/home/news/robert-jesse-award.jpg
 - assets/img/home/news/smart-sock.jpg
 - assets/img/home/news/va-health-incubator.png
 - assets/img/home/news/veteran-infertility.jpg
 - assets/img/home/news/veteran-rideshare.jpg
 - assets/img/home/news/vulnerable-veterans-rideshare.jpg
 - assets/img/home/news/xprize-pandemic-alliance.jpg
 - assets/img/home/va-ventures-logo.png
 - assets/img/home/vha-nci2i-logo.svg
 - assets/img/map-svg/default-pin.svg
 - assets/img/map-svg/usa.svg
 - assets/js/mustache-fellows-render.js
 - assets/js/nci2i-home-news.js
 - assets/js/vendors/lazy/jquery.lazy.min.js
 - assets/js/vendors/map-svg/jquery.mousewheel.min.js
 - assets/js/vendors/map-svg/jquery.nanoscroller.min.js
 - assets/js/vendors/map-svg/mapsvg.functions.js
 - assets/js/vendors/map-svg/mapsvg.min.js
 - assets/sass/vha/pages/home/accelerating-hci/_index.scss
 - assets/sass/vha/pages/home/featured-successes/_index.scss
 - assets/sass/vha/pages/home/masthead/_index.scss
 - assets/sass/vha/pages/home/national-centers-map/_index.scss
 - assets/sass/vha/pages/home/national-centers/_index.scss
 - assets/sass/vha/pages/home/news-and-events/_index.scss
 - assets/sass/vha/pages/home/our-mission/_index.scss
 - _includes/dark-header-sub.html
 - _includes/dark-header.html
 - _includes/header-sub.html
 - _includes/header.html
 - _site/assets/css/styles.css
 - _site/assets/js/functions.js
 - _site/assets/sass/vha/pages/home/_index.scss
 - _site/drhci/index.html
 - _site/ideas/index.html
 - _site/index.html
 - _site/nc4ti/index.html
 - _site/ncchi/index.html
 - _site/ncchi/our-team/index.html
 - _site/ncchi/publications/index.html
 - _site/va-ventures/index.html
 - assets/css/styles.css
 - assets/js/functions.js
 - assets/sass/vha/pages/home/_index.scss
 - index.html

ecstech-admin committed:
feat:initial build; includes & sass

ecstech-admin committed:
feat: add nci2i our mission

ecstech-admin committed:
feat: add successes; adjust mobile styles for mission

ecstech-admin committed:
feat: add accelerating-hci section

ecstech-admin committed:
feat: add natl ctrs map

ecstech-admin committed:
feat: add masthead; adjust mission styles

ecstech-admin committed:
feat: add national centers section

ecstech-admin committed:
feat: add news and events section

ecstech-admin committed:
bug: fix z-index; adjust padding; add video url

ecstech-admin committed:
feat: add nci2i to nav

ecstech-admin committed:
feat: add nci2i to nav; fix locations link

ecstech-admin committed:
feat: change ideas url

ecstech-admin committed:
bug: adjust padding

ecstech-admin committed:
fix: typos; missing js

ecstech-admin committed:
bug: fix misnamed scss files

ecstech-admin committed:
bug: rm misnamed scss files

ecstech-admin committed:
feat: add hero carousel photos

ecstech-admin committed:
bug: carousel height and btn styles

ecstech-admin committed:
feat: hero carousel test

ecstech-admin committed:
bug: add carousel styles

ecstech-admin committed:
bug: reset carousel img width

ecstech-admin committed:
Compress images

* [_site/assets/img/home/5g-tower.png] -249.6KB (-63%)
* [_site/assets/img/home/accelerating-innovation.png] -360.53KB (-59%)
* [_site/assets/img/home/drhci-logo.png] -16.85KB (-72%)
* [_site/assets/img/home/ideas-logo.png] -19.19KB (-72%)
* [_site/assets/img/home/ncchi-logo.png] -19.68KB (-71%)
* [_site/assets/img/home/nci2i-1.jpg] -8.24KB (-9%)
* [_site/assets/img/home/nci2i-10.jpg] -11.43KB (-9%)
* [_site/assets/img/home/nci2i-2.jpg] -5.62KB (-5%)
* [_site/assets/img/home/nci2i-3.jpg] -6.38KB (-6%)
* [_site/assets/img/home/nci2i-4.jpg] -19.79KB (-17%)
* [_site/assets/img/home/nci2i-5.jpg] -7.4KB (-6%)
* [_site/assets/img/home/nci2i-6.jpg] -5.71KB (-6%)
* [_site/assets/img/home/nci2i-7.jpg] -15.25KB (-28%)
* [_site/assets/img/home/nci2i-8.jpg] -4.94KB (-5%)
* [_site/assets/img/home/nci2i-9.jpg] -25.83KB (-22%)
* [_site/assets/img/home/nci2i-masthead.png] -753.48KB (-74%)
* [_site/assets/img/home/necie-logo.png] -18.94KB (-71%)
* [_site/assets/img/home/news/bring-people-ideas-together.jpg] -76.97KB (-75%)
* [_site/assets/img/home/news/capstone-project.jpg] -124.83KB (-72%)
* [_site/assets/img/home/news/disease-prevention-tools.jpg] -212.6KB (-72%)
* [_site/assets/img/home/news/face-of-innovation.jpg] -129.67KB (-72%)
* [_site/assets/img/home/news/first-innovation-center.jpg] -68.04KB (-79%)
* [_site/assets/img/home/news/jesse-award-osborne.jpg] -155.4KB (-57%)
* [_site/assets/img/home/news/legacy-partners.jpg] -96.94KB (-77%)
* [_site/assets/img/home/news/ncchi.jpg] -93.18KB (-76%)
* [_site/assets/img/home/news/personalized-healthcare.png] -277.86KB (-69%)
* [_site/assets/img/home/news/predictive-analytics.png] -338.91KB (-74%)
* [_site/assets/img/home/news/project-convergence.png] -345.68KB (-71%)
* [_site/assets/img/home/news/rideshare-for-veteran-employment.jpg] -86.04KB (-78%)
* [_site/assets/img/home/news/robert-jesse-award.jpg] -82.14KB (-40%)
* [_site/assets/img/home/news/smart-sock.jpg] -163.51KB (-72%)
* [_site/assets/img/home/news/va-health-incubator.png] -95.73KB (-54%)
* [_site/assets/img/home/news/veteran-infertility.jpg] -82.69KB (-80%)
* [_site/assets/img/home/news/veteran-rideshare.jpg] -100.36KB (-77%)
* [_site/assets/img/home/news/vulnerable-veterans-rideshare.jpg] -95.85KB (-68%)
* [_site/assets/img/home/news/xprize-pandemic-alliance.jpg] -204.62KB (-63%)
* [_site/assets/img/home/va-ventures-logo.png] -19.61KB (-71%)
* [assets/img/home/5g-tower.png] -249.6KB (-63%)
* [assets/img/home/accelerating-innovation.png] -360.53KB (-59%)
* [assets/img/home/drhci-logo.png] -16.85KB (-72%)
* [assets/img/home/ideas-logo.png] -19.19KB (-72%)
* [assets/img/home/ncchi-logo.png] -19.68KB (-71%)
* [assets/img/home/nci2i-1.jpg] -8.24KB (-9%)
* [assets/img/home/nci2i-10.jpg] -11.43KB (-9%)
* [assets/img/home/nci2i-2.jpg] -5.62KB (-5%)
* [assets/img/home/nci2i-3.jpg] -6.38KB (-6%)
* [assets/img/home/nci2i-4.jpg] -19.79KB (-17%)
* [assets/img/home/nci2i-5.jpg] -7.4KB (-6%)
* [assets/img/home/nci2i-6.jpg] -5.71KB (-6%)
* [assets/img/home/nci2i-7.jpg] -15.25KB (-28%)
* [assets/img/home/nci2i-8.jpg] -4.94KB (-5%)
* [assets/img/home/nci2i-9.jpg] -25.83KB (-22%)
* [assets/img/home/nci2i-masthead.png] -753.48KB (-74%)
* [assets/img/home/necie-logo.png] -18.94KB (-71%)
* [assets/img/home/news/bring-people-ideas-together.jpg] -76.97KB (-75%)
* [assets/img/home/news/capstone-project.jpg] -124.83KB (-72%)
* [assets/img/home/news/disease-prevention-tools.jpg] -212.6KB (-72%)
* [assets/img/home/news/face-of-innovation.jpg] -129.67KB (-72%)
* [assets/img/home/news/first-innovation-center.jpg] -68.04KB (-79%)
* [assets/img/home/news/jesse-award-osborne.jpg] -155.4KB (-57%)
* [assets/img/home/news/legacy-partners.jpg] -96.94KB (-77%)
* [assets/img/home/news/ncchi.jpg] -93.18KB (-76%)
* [assets/img/home/news/personalized-healthcare.png] -277.86KB (-69%)
* [assets/img/home/news/predictive-analytics.png] -338.91KB (-74%)
* [assets/img/home/news/project-convergence.png] -345.68KB (-71%)
* [assets/img/home/news/rideshare-for-veteran-employment.jpg] -86.04KB (-78%)
* [assets/img/home/news/robert-jesse-award.jpg] -82.14KB (-40%)
* [assets/img/home/news/smart-sock.jpg] -163.51KB (-72%)
* [assets/img/home/news/va-health-incubator.png] -95.73KB (-54%)
* [assets/img/home/news/veteran-infertility.jpg] -82.69KB (-80%)
* [assets/img/home/news/veteran-rideshare.jpg] -100.36KB (-77%)
* [assets/img/home/news/vulnerable-veterans-rideshare.jpg] -95.85KB (-68%)
* [assets/img/home/news/xprize-pandemic-alliance.jpg] -204.62KB (-63%)
* [assets/img/home/va-ventures-logo.png] -19.61KB (-71%)

ecstech-admin committed:
bug: correct image paths

ecstech-admin committed:
Merge branch 'feature/jls-6210515296/new-nci2i-page' of https://github.com/blackberggroup/VA-National-Centers into feature/jls-6210515296/new-nci2i-page

ecstech-admin committed:
bug: correct image paths

ecstech-admin committed:
bug: change IDEAS link from DRHCI to IDEAS

RyanCrux commented:
Couple things:

-  Arrow button on news stories is missing on mobile:
<img width="778" alt="Screenshot 2024-05-31 at 10 46 19 AM" src="https://github.com/blackberggroup/VA-National-Centers/assets/153299181/adfaa99f-1c2b-4556-b048-bcc0afe5f1b5">

- There is a small paragraph of text (only on mobile) that is missing under the news header
<img width="862" alt="Screenshot 2024-05-31 at 10 45 56 AM" src="https://github.com/blackberggroup/VA-National-Centers/assets/153299181/b2a9901e-ebb5-4511-a5bd-4ca2b70483fe">

-button styling under photo images doesn't match figma
<img width="1176" alt="Screenshot 2024-05-31 at 10 43 06 AM" src="https://github.com/blackberggroup/VA-National-Centers/assets/153299181/e8b18659-deaf-49be-bd0d-1d19162af681">

- This is probably because the locations may have been updated but the location copy info doesn't match source
<img width="1331" alt="Screenshot 2024-05-31 at 10 43 33 AM" src="https://github.com/blackberggroup/VA-National-Centers/assets/153299181/beafc641-872e-4196-b6ff-63d4d6ba0224">

- Arrow distance moves with screen size and eventually overlaps  
<img width="763" alt="Screenshot 2024-05-31 at 10 41 13 AM" src="https://github.com/blackberggroup/VA-National-Centers/assets/153299181/7e10b6f2-05e1-4bea-b311-ab14bb15fc43">

- too much space between sections here
<img width="814" alt="Screenshot 2024-05-31 at 10 44 30 AM" src="https://github.com/blackberggroup/VA-National-Centers/assets/153299181/f26dbd0e-9dde-422c-81b3-5c55df9ea5a0">

Overall looking really good! These pages are going to bring such a nice refresh!

ecstech-admin committed:
bug: hero slider styles; reduce mission margin-top

ecstech-admin committed:
bug: adjust news carousel break styles

ecstech-admin commented:
Hero carousel:
button styles corrected
rm extra padding between carousel and Our Mission

Map:
location edit: NECIE in Manchester NH (from Monday)
(note: static locations mirror popup style with name bold and other info not bold)

News:
Copy from Figma mobile is a mistake: held over from OAM design. 
Fixed styles so nav buttons are available at all breaks.

ecstech-admin committed:
feat: add alt tags to hero carousel

RyanCrux commented:
carousel photo placeholders are smaller than figma:
<img width="531" alt="Screenshot 2024-06-03 at 5 29 39 PM" src="https://github.com/blackberggroup/VA-National-Centers/assets/153299181/bfef17ac-2dd9-43e3-8482-de523dbb1105">

Map moves and cuts off when selecting location:
<img width="999" alt="Screenshot 2024-06-03 at 5 31 16 PM" src="https://github.com/blackberggroup/VA-National-Centers/assets/153299181/ea085d11-8327-4f41-8b62-ebd056502346">

forward/back buttons look better in many places but still overlap in one:
<img width="252" alt="Screenshot 2024-06-03 at 5 32 14 PM" src="https://github.com/blackberggroup/VA-National-Centers/assets/153299181/ec9e2fb5-aac6-42ab-9245-a19d1ef4a3fc">

ecstech-admin committed:
bug: fix hero carousel styles; fix news btn styles

will-bbg commented on _includes/home/masthead.html:
```diff
@@ -0,0 +1,76 @@
+        <div class="block block-color-2"></div>
+    </div>     
+</section>
+<!-- AWARDS - START
```
Remove commented code. This should not make it to production.

will-bbg commented on _includes/home/masthead.html:
```diff
@@ -0,0 +1,76 @@
+                                <source media="(min-width:640px)" srcset="assets/img/drhci/bubble-lines.svg">
+                                <img class="position-absolute circle-lines" src="assets/img/drhci/bubble-lines.svg" alt="Circle lines background">
+                            </picture>
+                            <!-- <img src="assets/img/home/nci2i-1.jpg" class="img-fluid" alt="NCI2I group photo" /> -->
```
Remove commented code. This should not make it to production.

will-bbg commented on _includes/home/our-mission.html:
```diff
@@ -0,0 +1,16 @@
+<!-- NCI2I Mission - START -->
+<!-- <section id="our-mission">
```
Remove commented code. This should not make it to production.

ecstech-admin committed:
feat: rm commented out content

will-bbg reviewed (changes requested):
### Masthead
- Masthead has an "h2". Make it an h1".
- On tablet, prev/next buttons appear out of nowhere. Please double check.
![Screenshot 2024-06-04 at 10 01 44 AM](https://github.com/blackberggroup/VA-National-Centers/assets/145696026/ea987ec3-1c52-44ec-a5ed-6f3a27ed1b64)

### Our Mission
- Reduce Our Mission section padding. Mockup shows 80px.
![Screenshot 2024-06-04 at 9 47 37 AM](https://github.com/blackberggroup/VA-National-Centers/assets/145696026/19980152-fe61-42ab-8b0d-068610473e3e)
- Our Mission should be an "h2" not an "h1"

### Logo Section
- Make logos link to websites
- Add alt tags to logo images
- Logo titles do not match the size in mockup
- Double check padding below logo titles for tablet and mobile. Currently does not match mockup

### News and Events
- Title should be an "h2" not an "h1"
- Let's remove the Dr. Osborne article. He is not longer with the program and was asked to be remove from NCCHI as well
- On mobile, buttons are too close to each other. Please see mockup.

ecstech-admin committed:
feat: make logos links

ecstech-admin committed:
bug: fix header order; H1/H2 sizing from Figma

ecstech-admin committed:
feat: add logo alt tags; make logo titles size/spacing match Figma

ecstech-admin committed:
feat: rm osborne news; change news section & title size; fix news btn spacing; rm comments

ecstech-admin committed:
bug: fix hero carousel buttons styles

ecstech-admin committed:
bug: clean up code, rm comments

ecstech-admin commented:
Made masthead title h1 (with h2 style) ((**This will need to be done on other pages (IDEAS, DRHCI, VA Ventures)**)
Added breaks for carousel arrow buttons and adjusted positioning as needed

Adjusted "Our Mission" padding to match Figma and made title h2 (with h1 style) ((**This will need to be done on other pages (IDEAS, DRHCI, VA Ventures**))

I added link + alt tags to logo images
Adjusted logo title sizes to match Figma size and padding

Made "News and Events" title h2
Removed 4th news article (Dr. Osborne)
Added breaks and adjusted positioning for carousel buttons
